src/tests/bulkConcepts/actions/bulkConcept.test.js

```diff
@@ -39,48 +37,6 @@ describe('Test suite for addBulkConcepts async actions', () => {
   afterEach(() => {
     moxios.uninstall(instance);
   });
-
-  it('should handle FETCH_BULK_CONCEPTS', () => {
-    moxios.wait(() => {
```

Comment:
Could you remind me on how the changes in this file are related to the fix?

Reply:
There was a bunch of duplicate code, in particular `fetchBulkConcepts` and `fetchFilteredConcepts` were doing the same thing when filtering was not being done.
Since `fetchBulkConcepts` is only used in this component, it needed to be deleted, along with its test.